bcda/web/api_test.go

```diff
@@ -517,49 +664,54 @@ func validateRequestHelper(endpoint string, s *APITestSuite) {
 		}
 	}
 
-	_, _, req = bulkRequestHelper(endpoint, "ExplanationOfBenefit")
+	_, _, req = bulkRequestHelper(endpoint, "ExplanationOfBenefit", "")
 	resourceTypes, err = validateRequest(req)
 	assert.Nil(s.T(), err)
 	assert.Equal(s.T(), 1, len(resourceTypes))
 	assert.Contains(s.T(), resourceTypes, "ExplanationOfBenefit")
 
-	_, _, req = bulkRequestHelper(endpoint, "Patient")
+	_, _, req = bulkRequestHelper(endpoint, "Patient", "")
 	resourceTypes, err = validateRequest(req)
 	assert.Nil(s.T(), err)
 	assert.Equal(s.T(), 1, len(resourceTypes))
 	assert.Contains(s.T(), resourceTypes, "Patient")
 
-	_, _, req = bulkRequestHelper(endpoint, "Coverage")
+	_, _, req = bulkRequestHelper(endpoint, "Coverage", "")
 	resourceTypes, err = validateRequest(req)
 	assert.Nil(s.T(), err)
 	assert.Equal(s.T(), 1, len(resourceTypes))
 	assert.Contains(s.T(), resourceTypes, "Coverage")
 
-	_, _, req = bulkRequestHelper(endpoint, "Practitioner")
+	_, _, req = bulkRequestHelper(endpoint, "Practitioner", "")
 	resourceTypes, err = validateRequest(req)
 	assert.Nil(s.T(), resourceTypes)
 	assert.Equal(s.T(), responseutils.Error, err.Issue[0].Severity)
 	assert.Equal(s.T(), responseutils.Exception, err.Issue[0].Code)
 	assert.Equal(s.T(), responseutils.RequestErr, err.Issue[0].Details.Coding[0].Display)
 
-	_, _, req = bulkRequestHelper(endpoint, "Patient,Patient")
+	_, _, req = bulkRequestHelper(endpoint, "Patient,Patient", "")
 	resourceTypes, err = validateRequest(req)
 	assert.Nil(s.T(), resourceTypes)
 	assert.Equal(s.T(), responseutils.Error, err.Issue[0].Severity)
 	assert.Equal(s.T(), responseutils.Exception, err.Issue[0].Code)
 	assert.Equal(s.T(), responseutils.RequestErr, err.Issue[0].Details.Coding[0].Display)
 }
 
-func bulkRequestHelper(endpoint, resourceType string) (string, func(http.ResponseWriter, *http.Request), *http.Request) {
+func bulkRequestHelper(endpoint, resourceType, since string) (string, func(http.ResponseWriter, *http.Request), *http.Request) {
 	var requestUrl string
 	var handlerFunc http.HandlerFunc
 	var req *http.Request
 
+	sinceRequestString := ""
+	if len(since) > 0 {
+		sinceRequestString = fmt.Sprintf("&_since=%s", since)
+	}
+
 	if endpoint == "Patient" {
 		if resourceType != "" {
-			requestUrl = fmt.Sprintf("/api/v1/%s/$export?_type=%s", endpoint, resourceType)
+			requestUrl = fmt.Sprintf("/api/v1/%s/$export?_type=%s%s", endpoint, resourceType, sinceRequestString)
 		} else {
-			requestUrl = fmt.Sprintf("/api/v1/%s/$export", endpoint)
+			requestUrl = fmt.Sprintf("/api/v1/%s/$export%s", endpoint, sinceRequestString)
```

Comment:
We probably don't want to omit the `?` and produce `/api/v1/Patient/$export&_since={date}`

Reply:
Good catch!